### ISSUE TITLE
Doesn't work when the same device connected to multiple bt-adapters.

### ISSUE THREAD

**Issue by x2es** [x2es/bt-dualboot#10]
I'm using Fedora Silverblue 37 and Windows 10 (I'll update the version shortly).  I was ultimately unsuccessful with using the script.  First, the script crashed because evidently the `info` file for my Logitech MX Master 3S does not contain the `LinkKey` key.  Not sure what this means, but I removed it as a bluetooth device and was able to proceed.

The issue that ultimately prevented me from was this:
```
WARNING: Following devices paired on Windows for multiple BT-adapters: <headphone mac>
```

I do have two bluetooth adapters in my desktop; one is built-in to the motherboard and scarcely works, the other is an external one that I use.  Indeed, my headphones have been, at different times, connected to each.   I tried just about everything, but could not figure out how to get this error to go away.  I wound up extracting the key by hand.

_Originally posted by @J3RN in https://github.com/x2es/bt-dualboot/issues/1#issuecomment-1382217564_

**Comment by retnag** [comment]
I've just had the exact same problem. This program checks for adapters (and subsequently paired devices) inside the `/var/bluetooth` directory on the linux side. For some reason an old adapters file-structure was in there, while the adapter was not actually used anymore (propably a leftower, from when I copied my installation over to another PC). Removing the directory for the old adapter solved the issue for me.

You can check which adapters are currently available with `bluetoothctl` and then issueing the `list` command. I checked the MAC address of my single (actually existing) controller, then I deleted all other folders in `/var/bluetooth`, so only the folder with the matching MAC address remained. Afterwards the warning disappeared, and syncing of my remaining (old) Bluetooth devices worked like a charm. Cheers.

**Comment by Monolitho** [comment]
> I've just had the exact same problem. This program checks for adapters (and subsequently paired devices) inside the `/var/bluetooth` directory on the linux side. For some reason an old adapters file-structure was in there, while the adapter was not actually used anymore (propably a leftower, from when I copied my installation over to another PC). Removing the directory for the old adapter solved the issue for me.
> 
> You can check which adapters are currently available with `bluetoothctl` and then issueing the `list` command. I checked the MAC address of my single (actually existing) controller, then I deleted all other folders in `/var/bluetooth`, so only the folder with the matching MAC address remained. Afterwards the warning disappeared, and syncing of my remaining (old) Bluetooth devices worked like a charm. Cheers.

Wait, is the var/bluetooth or var/lib/bluetooth? The former does not exist!

**Comment by Monolitho** [comment]
Nah...Not working. The warning is gone, but it still cannot get the issue fixed.

**Comment by Monolitho** [comment]
sudo   /media/disk1/programming/python/bin/bt-dualboot --sync-all --win /media/monolitho/Windows --backup
> BACKUP /media/monolitho/Windows/Windows/System32/config/SYSTEM to /var/backup/bt-dualboot/SYSTEM-2025-04-12--11-58-49

Syncing...
==========
 [A8:F5:E1:DE:A6:7B] OpenRun by Shokz
...done

**Comment by retnag** [comment]
> > I've just had the exact same problem. This program checks for adapters (and subsequently paired devices) inside the `/var/bluetooth` directory on the linux side. For some reason an old adapters file-structure was in there, while the adapter was not actually used anymore (propably a leftower, from when I copied my installation over to another PC). Removing the directory for the old adapter solved the issue for me.
> > You can check which adapters are currently available with `bluetoothctl` and then issueing the `list` command. I checked the MAC address of my single (actually existing) controller, then I deleted all other folders in `/var/bluetooth`, so only the folder with the matching MAC address remained. Afterwards the warning disappeared, and syncing of my remaining (old) Bluetooth devices worked like a charm. Cheers.
> 
> Wait, is the var/bluetooth or var/lib/bluetooth? The former does not exist!

sorry for the later reply, it probably depends on your distro. I'm on Manjaro, for me its /var/bluetooth, might be different for you.
